src/test/compile-fail/regions-outlives-projection-container-hrtb.rs

```diff
@@ -40,7 +40,6 @@ fn with_assoc<'a,'b>() {
     // We get an error because 'b:'a does not hold:
 
     let _: &'a WithHrAssoc<TheType<'b>> = loop { };
-    //~^ ERROR reference has a longer lifetime
```

Comment:
indeed, this error should not disappear -- when I investigated a bit, it appeared we were ICEing...?